src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Data.IO;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.ML
+{
+    public static class TextLoaderSaverCatalog
+    {
+        /// <summary>
+        /// Create a text reader.
+        /// </summary>
+        /// <param name="catalog">The catalog.</param>
+        /// <param name="args">The arguments to text reader, describing the data schema.</param>
+        /// <param name="dataSample">The optional data sample</param>
```

Comment:
data sample, or dataSource?

Reply:
It's a sample

---
In reply to: [224948938](https://github.com/dotnet/machinelearning/pull/1252#discussion_r224948938) [](ancestors = 224948938)